### ISSUE TITLE
[rainews.it] TGR broken, but works with youtube-dl

### ISSUE THREAD

**Issue by 4ricci** [yt-dlp/yt-dlp#1862]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2021.12.01**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Italy (but I don't know if it's accessible worldwide)

### Description

TGR videos from [rainews.it](https://www.rainews.it), available through `top-left menu -> "TGR" -> pick any region`, don't work with yt-dlp, but work fine with youtube-dl (version 2021.06.06).

I tried to fetch <https://www.rainews.it/tgr/emiliaromagna/notiziari/video/2021/12/ContentItem-d3afaafc-166f-40c3-9b34-1de5c736f39d.html> and I got `TypeError: '>' not supported between instances of 'NoneType' and 'int'`.

### Verbose log

```shell
[debug] Command-line config: ['-Uv', 'https://www.rainews.it/tgr/emiliaromagna/notiziari/video/2021/12/ContentItem-d3afaafc-166f-40c3-9b34-1de5c736f39d.html']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8 (No ANSI), err utf-8 (No ANSI), pref UTF-8
[debug] yt-dlp version 2021.12.01 [91f071af6]
[debug] Python version 3.10.0 (CPython 64bit) - Linux-5.10.82_1-x86_64-with-glibc2.32
[debug] exe versions: ffmpeg present (needs_adtstoasc,setts), ffprobe present
[debug] Optional libraries: mutagen, sqlite
[debug] Proxy map: {}
Latest version: 2021.12.01, Current version: 2021.12.01
yt-dlp is up to date (2021.12.01)
[debug] [Rai] Extracting URL: https://www.rainews.it/tgr/emiliaromagna/notiziari/video/2021/12/ContentItem-d3afaafc-166f-40c3-9b34-1de5c736f39d.html
[Rai] d3afaafc-166f-40c3-9b34-1de5c736f39d: Downloading webpage
[Rai] d3afaafc-166f-40c3-9b34-1de5c736f39d: Downloading video JSON
[Rai] d3afaafc-166f-40c3-9b34-1de5c736f39d: Downloading XML metadata for platform mon
[Rai] d3afaafc-166f-40c3-9b34-1de5c736f39d: Downloading m3u8 information
[Rai] d3afaafc-166f-40c3-9b34-1de5c736f39d: Downloading XML metadata for platform flash
[Rai] d3afaafc-166f-40c3-9b34-1de5c736f39d: Downloading XML metadata for platform native
ERROR: '>' not supported between instances of 'NoneType' and 'int'
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1329, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python3.10/site-packages/yt_dlp/YoutubeDL.py", line 1398, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/common.py", line 597, in extract
    ie_result = self._real_extract(url)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/rai.py", line 554, in _real_extract
    return self._extract_from_content_id(content_item_id, url)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/rai.py", line 487, in _extract_from_content_id
    relinker_info = self._extract_relinker_info(media['mediaUri'], content_id)
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/rai.py", line 103, in _extract_relinker_info
    formats.extend(self._create_http_urls(relinker_url, formats))
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/rai.py", line 193, in _create_http_urls
    fmt.update(get_format_info(q))
  File "/usr/lib/python3.10/site-packages/yt_dlp/extractor/rai.py", line 146, in get_format_info
    if br > 300:
TypeError: '>' not supported between instances of 'NoneType' and 'int'
```

**Comment by pukkandan** [comment]
@nixxo

**Comment by nixxo** [comment]
it was a bug in the method that generated direct mp4 link because it extracted infos from the m3u8 manifest listand having audio only formats in the list created issues.

Also by running the test I discovered that the website changed a little bit and added a little tweak to the alt_title extraction.
Plus I also changed one of the tests because the page was no longer available.

I put everything in the same PR, I'll separate if necessary.
